sources/views/contacts.js

```diff
@@ -1,47 +1,62 @@
-import { JetView } from "webix-jet";
-import { contacts } from "../models/contacts.js";
+import {JetView} from "webix-jet";
+import {contacts} from "../models/contacts";
 
-import ContactInfo from "./contact-info.js";
 import noPhoto from "../assets/img/noPhoto.png";
 
 export default class Contact extends JetView {
 	config() {
-		const _ = this.app.getService("locale")._;
 		const usersList = {
 			view: "list",
 			localId: "userList",
 			width: 300,
 			select: true,
 			scroll: false,
-			template: obj => {
-				return `
+			template: obj => `
 						<div class="user-list--grid">
 							<div class="user-list-photo"><img src=${obj.Photo || noPhoto} alt="User photo" /></div>
 							<div><span>${obj.value}</span><br /><span>${obj.Company}</span></div>
```

Comment:
`obj.value` is not updated after editing. 
It will remain here and in options. 
You can test it as follows:
- open editing from for any contact
- change name, save it
- data will be sent to server, but name in the list won't change

Reply:
Fixed.